docs/src/literate/helmholtz.jl

```diff
@@ -26,7 +26,7 @@
 # \int \nabla δu \cdot \nabla u d\Omega
 # + \int δu \cdot u d\Omega
 # - \int δu \cdot f d\Omega
-# + \int δu \cdot (n \cdot \nabla u - g_2) d\Gamma_2
+# - \int δu \cdot g_2 d\Gamma_2 = 0 \forall δu
```

Comment:
I think there is no need for the \cdot here, is there? Seems to me like these are all scalars (same for the two terms above).

Reply:
Maybe it would be easier to read if it was `int_{\Gamma_2}` as well (and similar for the other terms), once we're on changing it anyways. What do you think?